README.md

```diff
@@ -131,39 +126,19 @@ var creator_ip = '127.0.0.1'
 var creator_wakeword_base_port = 60001;
 ```
 
-#### Load Proto files
-
-``` javascript
-var driverProtoBuilder = protoBuf.loadProtoFile({
-  root: PROTO_PATH, 
-  file: 'matrix_io/malos/v1/driver.proto'
-})
-var ioProtoBuilder = protoBuf.loadProtoFile({
-  root: PROTO_PATH, 
-  file: 'matrix_io/malos/v1/io.proto'
-})
-
-var matrix = {
-  malos: {
-    v1: {
-      driver: driverProtoBuilder.build("matrix_io.malos.v1.driver"),
-      io: ioProtoBuilder.build("matrix_io.malos.v1.io")
-    }
-  }
-}
-```
-
 #### Config and start wakeupword service
 
 ``` javascript
 function startWakeUpRecognition(){
   console.log('<== config wakeword recognition..')
-  var wakeword_config = new matrix.malos.v1.io.WakeWordParams;
-  wakeword_config.set_wake_word("MIA");
-  wakeword_config.set_lm_path("/home/pi/assets/9854.lm");
-  wakeword_config.set_dic_path("/home/pi/assets/9854.dic");
-  wakeword_config.set_channel(matrix.malos.v1.io.WakeWordParams.MicChannel.channel8);
-  wakeword_config.set_enable_verbose(false)
+  var wakeword_config = matrix_io.malos.v1.io.WakeWordParams.create({
+    wakeWord: 'MIA',
+    lmPath: LM_PATH,
+    dicPath: DIC_PATH,
+    channel: matrix_io.malos.v1.io.WakeWordParams.MicChannel.channel8,
+    enableVerbose: false
+  });
+
```

Comment:
I also like this api more, even when the camel casing gave caused me trouble at first.